gensim/corpora/ucicorpus.py

```diff
@@ -15,7 +15,7 @@
 from gensim import utils
 from gensim.corpora import Dictionary
 from gensim.corpora import IndexedCorpus
-from gensim.matutils import MmReader
```

Comment:
Please move `MmReader` to `matutils`  (yes, this is wired, but needed for backward compatibility). We'll move it to `corpora` with major release.

Reply:
ok done.  I kept the cython version as "gensim.corpora._mmreader".  Let me know if you want this renamed to gensim.corpora.mmreader or something else.